cve_bin_tool/data_sources/curl_source.py

```diff
@@ -40,12 +42,14 @@ def __init__(self, error_mode=ErrorMode.TruncTrace):
         self.vulnerability_data = []
 
     async def get_cve_data(self):
+        """Get cURL vulnerability data. Fetches the cURL vulnerability data and retrieves a list of affected data. Returns: Tuple[Union[None, Any], str]: A tuple containing the affected data and the source name."""
```

Comment:
```suggestion
        """Get cURL vulnerability data. Fetches the cURL vulnerability data and retrieves a list of 
        affected data. 
        
        Returns: 
        Tuple[Union[None, Any], str]: A tuple containing the affected data and the source name."""
```

This line is really long and doesn't particularly need to be.  Could you try to get it to fit within the pep8 79-character line recommendation?

Reply:
(you probably don't actually want to use my recommendation as is, I'm just trying to give you the idea but the web interface doesn't actually show me when I hit 79 characters)